source/administration/performance-alerting-guide.rst

```diff
@@ -0,0 +1,121 @@
+
+.. image:: ../images/perf-4.png
+
+Fill in a message if you would like to add more context to the alert.
+
+By default, the alerts are configured to check the average of a chart over the last minute to see if that value is above a threshold. If it’s above the threshold, the alert will be triggered. Since it’s an average over the last minute, small spikes that go past the threshold won’t necessarily cause an alert. This helps prevent false positives that would result from natural spikes in usage. The alert state for each chart is evaluated every minute.
+
+The sections below describe each chart in more detail.
+
+CPU Utilization Rate
+---------------------
+
+CPU Utilization Rate is fairly straightforward. It tracks the CPU usage of the app servers as a percentage. The maximum percentage is based on the number of CPU cores or vCPUs your app server has. For example, if you have four CPU cores and your app server was at 100% utilization rate on all four cores, the graph would show 400% for that app server.
+
+It’s best to set the alert threshold based on your average CPU utilization and how many cores/vCPUs your app servers have. Take a look at the chart over the last seven days. You’ll want to set the threshold somewhere between your maximum CPU usage (cores * 100) and the CPU usage you see. The lower you set the threshold, the more alerts, and therefore the more false positives, you will get. Set the threshold too high and you may not receive an alert for an incident until it’s gotten worse. The same principle applies to all alerts, regardless of the chart.
+
+For example, on our daily build server that much of the team and community works out of, we have the threshold set to 15%:
+
+.. image:: ../images/perf-5.png
+
+This is well below our maximum CPU usage as we don’t have a high load. It’s also above our average usage at peak times, so we will get alerts if we start experiencing irregularly high CPU usage.
+
+Memory Usage
+-------------
+
+Memory Usage is also straightforward. It involves simply tracking the megabytes of RAM that your app servers are using. You’ll want to set the threshold much like you did for the CPU Utilization Rate: below maximum available memory and above your average usage during peak times.
+
+Here’s how we have the alert set on our daily build server: 
+
+.. image:: ../images/perf-6.png
+
+Number of Goroutines
+---------------------
+
+If you’re not familiar with Go and goroutines, this chart might not make much sense to you upfront. Goroutines are functions or methods that run concurrently with other functions and methods. You can think of them as lightweight threads with low-creation costs. A rising number of goroutines can be a good measure of the performance of your app servers. If they begin to continuously climb, that indicates your app server can’t keep up and is creating goroutines faster than they can complete their tasks and stop.
```

Comment:
Remove `If you’re not familiar with Go and goroutines, this chart might not make much sense to you upfront.` as it doesn't add value to the guide.

Reply:
```
You can think of them as lightweight threads with low-creation costs. A rising number of goroutines can be a good measure of the performance of your app servers. If they begin to continuously climb, that indicates your app server can’t keep up and is creating goroutines faster than they can complete their tasks and stop.
```

to

```
Goroutines are like lightweight threads with low-creation costs. A rising number of goroutines can be a good measure of the performance of your app servers. A continuous increase indicates your app server can't keep up and is creating goroutines faster than they can complete their tasks and stop.
```